Externals_CLM.cfg

```diff
@@ -1,8 +1,8 @@
 [fates]
 local_path = src/fates
 protocol = git
-repo_url = https://github.com/NGEET/fates
-tag = sci.1.67.2_api.27.0.0
+repo_url = https://github.com/rgknox/fates
```

Comment:
This will need to be updated to a tag before coming into main-dev (as per usual...)

Reply:
yup, will update at time of integration since this is a synchronized PR